ui/StatusQ/src/StatusQ/Controls/StatusBinaryRadioControl.qml

```diff
@@ -0,0 +1,39 @@
+import QtQuick 2.14
```

Comment:
We don't need this control at all; standard radio button handles that directly.

Radio buttons are mutually exclusive (binary) by default and if there's a need/pattern for such control, they can be always placed inside a ButtonGroup or GroupBox

Reply:
We had these functionality duplicated multiple times, it also often had bugs or slightly different designs (due to duplication), this single abstraction makes it consistent and without having to worry about details. (it's also a poc/test for other more complex controls down the line). I don't see GroupBox removing this duplication

Reply:
Yeah I got the general point; but I don't see anything here that could not be solved using a GroupBox. Also a Control should start with Control, not Column. `property var onToggle` is also something that should be replaced by a signal, etc